packages/gluestick/src/commands/test/test.js

```diff
@@ -62,27 +62,18 @@ const getDebugDefaultConfig = (aliases, webpackRules) => {
   argv.push(...getJestDefaultConfig(aliases, webpackRules));
   argv.push('-i');
   argv.push('--watch');
-  return argv;
-};
-
-const createArgs = (defaultArgs, options) => {
-  const argv = [].concat(defaultArgs);
-  const { coverage, watch, pattern } = options;
-
-  if (coverage) {
-    argv.push('--coverage');
-  }
-  if (watch) {
-    argv.push('--watch');
-  }
-  if (pattern) {
-    argv.push(pattern);
-  }
-
-  return argv;
+  // Exclude those options to avoid dublication.
+  const optionsToExclude = ['-D', '--debug-test', '-i', '--runInBand', '--watch', '--config', '-c'];
+  return argv.concat(
+    options.filter((option: string): boolean => {
+      return optionsToExclude.findIndex((optionToExclude: string): boolean => {
+        return new RegExp(`^${optionToExclude}.*`).test(option);
```

Comment:
I would log an info message to the user. Something like "option X is always set by default on debug mode".